dokka-integration-tests/gradle/build.gradle.kts

```diff
@@ -1,43 +1,208 @@
-import org.jetbrains.kotlin.gradle.dsl.ExplicitApiMode
-
 /*
  * Copyright 2014-2023 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
  */
+import org.gradle.api.tasks.testing.logging.TestExceptionFormat.FULL
+import org.gradle.api.tasks.testing.logging.TestLogEvent.FAILED
+import org.gradle.api.tasks.testing.logging.TestLogEvent.SKIPPED
+import org.gradle.language.base.plugins.LifecycleBasePlugin.VERIFICATION_GROUP
+import org.jetbrains.kotlin.gradle.dsl.ExplicitApiMode.Disabled
 
 plugins {
-    id("dokkabuild.test-integration")
+    id("dokkabuild.kotlin-jvm")
+    `test-suite-base`
```

Comment:
What is the difference between `jvm-test-suite` and `test-suite-base` plugins and why we use `base` one?